kolibri/plugins/setup_wizard/assets/src/state/actions.js

```diff
@@ -1,12 +1,12 @@
 import { DeviceProvisionResource } from 'kolibri.resources';
 import { kolibriLogin, handleApiError } from 'kolibri.coreVue.vuex.actions';
 
-export function provisionDevice(store, superuser, facility, preset, language_id) {
+export function provisionDevice(store, superuser, facility, preset, language_code) {
```

Comment:
The switch to `language_id` was intentional: https://github.com/christianmemije/kolibri/commit/868a24182e02b3dc8669e2beeae89734b5de474b

The mistake was not changing the backend.

Have made a PR that does that: https://github.com/learningequality/kolibri/pull/2092